main.go

```diff
@@ -1,43 +1,69 @@
 package main
 
 import (
+	"context"
 	"fmt"
 	"log"
 	"net/http"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
 	"time"
 
-	"github.com/alphagov/paas-prometheus-exporter/exporter"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+
+	"github.com/alphagov/paas-prometheus-exporter/app"
+	"github.com/alphagov/paas-prometheus-exporter/cf"
+	"github.com/alphagov/paas-prometheus-exporter/service"
+
 	"github.com/cloudfoundry-community/go-cfclient"
 	"github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"gopkg.in/alecthomas/kingpin.v2"
 )
 
 var (
 	version            = "0.0.3"
```

Comment:
Please bump this version as part of this PR :)

Reply:
👍